.github/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 <h3>New features since last release</h3>
 
+* Added the ``IQPEmbeddings`` template, which encodes inputs into the diagonal gates of an 
+  IQP circuit. 
+  [(#605)](https://github.com/XanaduAI/pennylane/pull/605)
+
 * The ``templates.broadcast`` function can now take custom patterns.
   [(#603)](https://github.com/XanaduAI/pennylane/pull/603)
```

Comment:
Do you want to add examples of each of these :)

Reply:
We've started making richer changelog messages now that it's embedded in the documentation

Reply:
I added code and image. Note that the image will only show once the doc version is live!